### ISSUE TITLE
Restriction on demo video

### ISSUE THREAD

**Issue by lm5lm5** [nus-cs2103-AY1920S2/forum#125]
Is there any restriction on the product demo video apart from the size of the video, like the maximal time of the video, the minimal time each team member should participate in the demo etc?

**Comment by okkhoy** [comment]
Think about it as a normal demo.
All conditions specified [here](https://nus-cs2103-ay1920s2.github.io/website/admin/tp-deliverables.html#deliverable-demo) holds

**Comment by chrisjwelly** [comment]
Hi Prof, the website says: “Each person should do a fair share of the demo. However, it's OK for one member to does all the typing.”

However, due to the situation and possible difficulties in coordinating, would it be okay if only 1 or 2 people does the speaking as well?

**Comment by okkhoy** [comment]
No no.. I still want all team members to talk. 
Think of it as a regular demo where each of you presents their part. 
I think just logging in to one zoom meeting, recording the meeting and uploading the mp4 is good enough.

I want to reiterate:
Don't spend too much time or stress over it. As long as you show what your app does and how does it solve the target user's problem it is a good video already 🙂
Don't waste time on editing.

**Comment by raivatshah** [comment]
Thanks for clarifying prof!

**Comment by aakanksha-rai** [comment]
Prof @okkhoy , do our faces also have to be visible in the demo or is a screen recording of the application with our voices explaining what's going on enough?

**Comment by raivatshah** [comment]
@aakanksha-rai I don't think so. Just the screen recording with the app and our voices should be fine I think.

**Comment by okkhoy** [comment]
Not mandatory; but if you show yourself, the evaluator will know who is talking.
However, introducing yourself is a must.
